pkg/kubelet/dockertools/docker.go

```diff
@@ -276,16 +277,19 @@ func LogSymlink(containerLogsDir, podFullName, containerName, dockerId string) s
 	return path.Join(containerLogsDir, fmt.Sprintf("%s_%s-%s.%s", podFullName, containerName, dockerId, LogSuffix))
 }
 
-// Get a *docker.Client, either using the endpoint passed in, or using
+// Get a *dockerapi.Client, either using the endpoint passed in, or using
 // DOCKER_HOST, DOCKER_TLS_VERIFY, and DOCKER_CERT path per their spec
-func getDockerClient(dockerEndpoint string) (*docker.Client, error) {
+func getDockerClient(dockerEndpoint string) (*dockerapi.Client, error) {
 	if len(dockerEndpoint) > 0 {
 		glog.Infof("Connecting to docker on %s", dockerEndpoint)
-		return docker.NewClient(dockerEndpoint)
+		return dockerapi.NewClient(dockerEndpoint, "", nil, nil)
```

Comment:
From engine-api doc, looks like if the version number is empty string, the docker client assume you are talking to the latest API. In this case, do we have issues to talk to docker daemon 1.8.3 release?